starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,71 @@
+div{
+    border:1px red solid;
+    
+}
+
+body {
+    font-family: Helvetica;
+}
+
+img {
+    width: 100%;
+}
+
+.container #info {
+    width: 60%
+    padding-left: 200px;
+    position: centered;
```

Comment:
This is not working as you expect. There are many ways to center elements, but usually you are going to use 
```
text-align: center; /* (inline elements) */
```
or 
```
margin: 0 auto; /* (block element) */ 
```
Here you have an amazing guide: https://css-tricks.com/centering-css-complete-guide/